src/components/AuthTextField/AuthTextField.tsx

```diff
@@ -18,22 +23,45 @@ const AuthTextField: FC<InputProps> = ({
   value,
   onChange,
   placeholder,
+  togglePasswordVisibility = undefined,
+  handleKeyDown = undefined,
+  showPassword = undefined,
+  showEyeIcon = false,
```

Comment:
The setting of these default prop values are unnecessary. `togglePasswordVisibility, handleKeyDown, showPassword` are all set to undefined by default. However, in TypeScript, they are undefined if you don't pass these props.

Reply:
Ok, I will make a change on it.

Reply:
However, when I delete the ```undefined```, there comes an error: ```propType "handleKeyDown" is not required, but has no corresponding default argument value.eslint[react/require-default-props](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/require-default-props.md)```

I may remain those values assigned to my props since all four of them are optional for the Textfield